lib/ecto/migration/boot_migrator.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Ecto.Migration.BootMigrator do
+  @moduledoc """
+  A process you can add to your Application Supervisor to run migrations. This will attempt to run migrations, and then silently shut its self down.
+
+  Add the following to the top of your application children spec:
+
+    {Ecto.Migration.BootMigrator, repos: Application.fetch_env!(:my_app, :ecto_repos)}
```

Comment:
Probably should add a note about timeouts, since the default behaviour here is different than `mix ecto.migrate` which uses `timeout: :infinity`